tmol/tests/data/__init__.py

```diff
@@ -76,53 +75,25 @@ def disulfide_res():
 
 
 @pytest.fixture()
-def cst_system():
-    from tmol.system.io import read_pdb
-
-    return read_pdb(pdb.data["6DMZ_A.pdb"])
-
-
-@pytest.fixture()
-def cst_csts():
-    from numpy import load
-
-    return dict(load(os.path.dirname(__file__) + "/constraints/6DMZ_A.npz"))
-
+def water_box_res():
+    from tmol.system.io import ResidueReader
 
-@pytest.fixture(scope="session")
-def ubq_rosetta_baseline():
-    # TODO ubq baseline does *not* contain the same conformation as ubq_pdb
-    return rosetta_baseline.data["1ubq"]
+    return ResidueReader.get_default().parse_pdb(pdb.data["water_box"])
 
 
 @pytest.fixture(scope="session")
 def systems_bysize():
     from tmol.system.io import read_pdb
 
     return {
-        40: read_pdb(pdb.data["bysize_040_res_5uoi.pdb"]),
-        75: read_pdb(pdb.data["bysize_075_res_2mtq.pdb"]),
-        150: read_pdb(pdb.data["bysize_150_res_5yzf.pdb"]),
-        300: read_pdb(pdb.data["bysize_300_res_6f8b.pdb"]),
-        600: read_pdb(pdb.data["bysize_600_res_5m4a.pdb"]),
+        40: pdb.data["bysize_040_res_5uoi.pdb"],
+        75: pdb.data["bysize_075_res_2mtq.pdb"],
+        150: pdb.data["bysize_150_res_5yzf.pdb"],
+        300: pdb.data["bysize_300_res_6f8b.pdb"],
+        600: pdb.data["bysize_600_res_5m4a.pdb"],
```

Comment:
Interesting - do we want to use these at all for the score term test?

Reply:
I don't know that we need to.  This is used currently to show runtime scaling as a function of protein size.

Reply:
I was thinking of using these as a more realistic set of data for the 'jagged' score term test, but I guess different terms may need different inputs to ensure they're tested properly (disulfide, for example).